release-notes.md

```diff
@@ -10,6 +10,16 @@ The Adobe Experience Platform Mobile SDK will soon be available in Swift as an i
 
 [Sign up](https://forms.microsoft.com/Pages/ResponsePage.aspx?id=Wht7-jR7h0OUrtLBeN7O4UJN9zAhIEhJr3PBfyMf9wdUQTI2S0pMVEVYS1k3UUNJVDNDWlRUTFk4Qi4u) for a free developer preview.
 
+### September 10, 2020
+
+#### iOS Core 2.7.4
+
+* Fixed a crash issue happened on `ACPExtensionAPI`.
```

Comment:
Fixed an intermitted issue where third party extensions may experience crashes after unregistration when using the ACPExtensionAPI